client/src/assets/styles/projector.scss

```
@@ -11,13 +11,26 @@
         line-height: 1.1em;
     }
 
+    .slidetitle > h1,
+    .slidetitle > h1 *,
+    h1.projector_h1,
+    h1.projector_h1 * {
+        font-family: $font-projectorH1;
+    }
```

```suggestion
    .slidetitle > h1,
    h1.projector_h1 {
        &, * {
            font-family: $font-projectorH1;
        }
    }
```

I think, this is equivalent. Maybe just a detail, but in my opinion it is more readable.